OneSignalSDK/onesignal/src/main/java/com/onesignal/OneSignal.java

```diff
@@ -2093,7 +2093,7 @@ static boolean startOrResumeApp(Context inContext) {
       Intent launchIntent = inContext.getPackageManager().getLaunchIntentForPackage(inContext.getPackageName());
       // Make sure we have a launcher intent.
       if (launchIntent != null) {
-         launchIntent.setFlags(Intent.FLAG_ACTIVITY_REORDER_TO_FRONT | Intent.FLAG_ACTIVITY_NEW_TASK);
+         launchIntent.setFlags(Intent.FLAG_ACTIVITY_CLEAR_TOP | Intent.FLAG_ACTIVITY_NEW_TASK);
```

Comment:
Will these flag result in a new Activity each time? Or will it be reused if it already exists?
For wrapper SDK we need to the flag to result in using existing instances, since it will create multiple instances of the framework environment and handles. Such as with react native and Unity.

Reply:
If it already exist will be reused, for example if i have A->B->C and i launched A then the stack will be the A already there